dev/io.openliberty.org.jboss.resteasy.common/bnd.bnd

```diff
@@ -52,9 +50,8 @@ Service-Component: \
     properties:="resources=${app-resources}"
 
 Export-Package: \
-  com.ibm.websphere.jaxrs20.multipart;thread-context=true, \
+  com.ibm.websphere.jaxrs20.multipart;version=10;thread-context=true, \
```

Comment:
@jhanders34 I don't like what I had to do here, what's the best way to make sure the exported package fits in [10,11)?

Reply:
What you have is correct

Reply:
You also should update the package-info.java file to be 10.0